scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -284,24 +391,16 @@ version_gt() {
 
 summary()
 {
-    kernel_version=$(uname -r)
-
     echo
-    echo Installation completed successfully.
+    echo "Installation completed successfully."
     echo
-    echo Next steps:
-    echo "Add OpenCL users to the video group: 'sudo usermod -a -G video USERNAME'"
-    echo "   e.g. if the user running OpenCL host applications is foo, run: sudo usermod -a -G video foo"
-    echo "   Current user has been already added to the video group"
+    echo "Next steps:"
+    echo "Add OpenCL users to the video and render group: 'sudo usermod -a -G video,render USERNAME'"
```

Comment:
If one of groups does not exist - error will be returned. I think this instruction can be slightly rephrased.

Reply:
Currently you can see function that trying to add user in each group separately
https://github.com/openvinotoolkit/openvino/blob/cd7114179065a7328b5f696b9bb6e1b19cb9f206/scripts/install_dependencies/install_NEO_OCL_driver.sh#L392

Reply:
Yes, it is about the message being printed for users - this command will not work in some environments, e.g. Ubuntu 18.

Reply:
BTW, automatic adding to group does not work when running as root (not sudo):
```
Adding  to the video group...
usermod: user '' does not exist
WARNING: unable to add  to the video group
Adding  to the render group...
usermod: group 'render' does not exist
WARNING: unable to add  to the render group
```